code/modules/mob/living/silicon/emote.dm

```diff
@@ -0,0 +1,156 @@
+/datum/emote/silicon
+	mob_type_allowed_typecache = list(/mob/living/silicon/robot)
```

Comment:
is it possible that, instead of doing this, we can add `/mob/living/silicon/robot` to the mob_type typecache on `/datum/emote`, and then manually exclude it from the ones we don't want borgs using?

Reply:
emote typecaches are populated in new, /datum/emote's is just every /mob, but that's just for *help anyway